client/src/app/components/header/header.component.html

```diff
@@ -34,6 +34,19 @@
 
   <!-- User menu -->
   <mat-menu #userMenu="matMenu" xPosition="before">
-    <button mat-menu-item (click)="logout()">Logout</button>
+    <div class="avatar-wrapper avatar-wrapper-padding" *ngIf="user">
```

Comment:
Maybe instead of checking `*ngIf="user"` three times here, we could add it to L36 (or create a container right underneath?